runelite-api/src/main/java/net/runelite/api/ScriptID.java

```diff
@@ -233,4 +233,16 @@
 	 */
 	@ScriptArguments(integer = 2)
 	public static final int TOPLEVEL_RESIZE = 909;
+
+	/**
+	 * Called when the friends list is updated
```

Comment:
Since the meaning the parameters is known, they should be listed in the javadoc. See some of the earlier ScriptID entries for examples.